brotab/extension/background.js

```diff
@@ -102,16 +113,58 @@ class FirefoxTabs extends BrowserTabs {
   getBrowserName() {
       return "firefox";
   }
+
+  activate(tab_id) {
+    this._browser.tabs.update(tab_id, {'active': true});
```

Comment:
@naeloob  The documentation says "focused" but you use "active". Is that a typo?